3604/2/modules/checkin_command.rb

```diff
@@ -0,0 +1,57 @@
+module CheckinCommand
+  def checkin!(*)
+    return respond_if_are_not_registered unless redis.get(user_id_telegram)
+    return respond_if_session_checkin if session[:status] == 'checkin'
+    session[:time_checkin] = Time.now.to_i
+    session[:status] = 'checkin'
+    diolog_about_photo_checkin
+  end
+
+  def respond_if_session_checkin
+    respond_with :message, text: 'First make a command /checkout'
+  end
+
+  def diolog_about_photo_checkin
```

Comment:
typo: diAlog)

Reply:
usually we try to avoid using prepositions in method names. Ex. photo_checkin_dialog. but here your variant even more readable. you can leave any of them. just remember this fact